Documentation/templates/strings/flat-swift4.md

```diff
@@ -28,15 +28,19 @@ You can customize some elements of this template by overriding the following par
 **Extract:**
 
 ```swift
-enum L10n {
+internal enum L10n {
   /// Some alert body there
-  case alertMessage
+  internal static let alertMessage = L10n.tr("Localizable", "alert__message")
   /// Title of the alert
-  case alertTitle
+  internal static let alertTitle = L10n.tr("Localizable", "alert__title")
   /// You have %d apples
-  case applesCount(Int)
+  internal static func applesCount(_ p1: Int) -> String {
+    return L10n.tr("Localizable", "apples.count", p1)
+  }
   /// Those %d bananas belong to %@.
-  case bananasOwner(Int, String)
+  internal static func bananasOwner(_ p1: Int, _ p2: Any) -> String {
+    return L10n.tr("Localizable", "bananas.owner", p1, String(describing: p2))
+  }
```

Comment:
We should find a way to ensure those get updated regularly, too easy to forget… (Danger rule probably easier, script auto-updating the doc when generating outputs would probably be too complicated if we want to omit stuff for brevity in those docs)

Reply:
Danger script sounds like a good idea, anytime the template is changed, and not the corresponding docs --> warning.